runners/spark/pom.xml

```diff
@@ -112,29 +112,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-streaming-kafka_2.10</artifactId>
```

Comment:
Opened [BEAM-785](https://issues.apache.org/jira/browse/BEAM-785) as a separate issue, to follow after this PR.
Probably most of the tests would be ignored at first (due to lack of support in triggers, accumulations, etc.) but it'd be nice to have something to start with.